src/roboteq_controller.cpp

```diff
@@ -0,0 +1,296 @@
+#include <memory>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "roboteq_controller.hpp"
+#include "i_comm.hpp"
+
+namespace roboteq {
+
+    roboteq_controller::roboteq_controller(std::unique_ptr<i_comm> &&comm) : _comm(std::move(comm)) {
+
+    }
+
+     // set methods
+    int32_t roboteq_controller::SetPosition(int32_t position, uint8_t channel){
```

Comment:
should all the sets not just return a bool? to indicate success or not? i don't think any of the runtime commands actually return a value